egs/chime4/s5_1ch/local/clean_chime4_format_data.sh

```diff
@@ -0,0 +1,85 @@
+#!/bin/bash
+
+# Copyright 2012  Microsoft Corporation  Johns Hopkins University (Author: Daniel Povey)
```

Comment:
These are originally from wsj/s5/local/wsj_format_data.sh (long time ago)
How should I modify the copyright?
Also I noticed that wsj/s5/local/wsj_format_data.sh becomes simple by using "utils/validate_lang.pl" etc.
Should I change it like that?
This is similar to @jtrmal 's suggestion below.